content/en/providers/operations-manuals/howto01_using_igtf_ca_distribution/_index.md

```diff
@@ -0,0 +1,322 @@
+
+- Install the EUGridPMA PGP key for apt:
+
+```shell
+wget -q -O - \
+  https://dist.eugridpma.info/distribution/igtf/current/GPG-KEY-EUGridPMA-RPM-3 \
+  | apt-key add -
+```
+
+- Add the following line to your sources.list file for APT:
+
+```shell
+#### EGI Trust Anchor Distribution ####
+deb http://repository.egi.eu/sw/production/cas/1/current egi-igtf core
+```
+
+- Populate the cache and install the meta-package
+
+```shell
+apt-get update apt-get install ca-policy-egi-core
+```
+
+### Using the distribution on other (non-RPM) platforms
+
+The trust anchors are provided also as simple 'tar-balls' for installation on
+other platforms. Since there is no dependency management in this case, please
+review the release notes carefully for any security issues or withdrawn CAs. The
+tar files can be found in the EGI repository at
+
+```text
+http://repository.egi.eu/sw/production/cas/1/current/tgz/
+```
+
+Once you have downloaded the directory, you can unpack all the CA tar,gz as
+follows to your certificate directory:
+
+```shell
+for tgz in $(ls  <ca download dir>); do
+  tar xzf <ca download dir>/$tgz --strip-components=1 \
+    -C /etc/grid-security/certificates
+done
+```
+
+### Installing the distribution using Quattor
+
+Quattor templates are povided as drop-in replacements for both QWG and CDB
+installations. Update your software repository (re-generating the repository
+templates as needed) and obtain the new CA templates from:
+
+- `http://repository.egi.eu/sw/production/cas/1/current/meta/ca-policy-egi-core.tpl`
+  for QWG
+- `http://repository.egi.eu/sw/production/cas/1/current/meta/pro_software_meta_ca_policy_egi_core.tpl`
+  for CDB
+
+Make sure to mirror (or refer to) the new repository at
+`http://repository.egi.eu/sw/production/cas/1/current/` and create the
+appropriate repository definition file.
+
+For WLCG sites that are migrating from the lcg-CA package: the WLCG policy
+companion of the EGI templates can be found at
+[QWG](http://lcg-ca.web.cern.ch/lcg-ca/distribution/current/meta/ca-policy-lcg.tpl)
+and
+[CDB](http://lcg-ca.web.cern.ch/lcg-ca/distribution/current/meta/pro_software_meta_ca_policy_lcg.tpl)
+and can be included in the profile in parallel with the EGI core template. All
+packages needed are also included in the EGI repository, so only a single
+repository reference is necessary.
+
+## Trust Anchor structure for differentiated (collaborative) assurance
+
+EGI is moving towards support for 'differentiated' or collaborative assurance,
+where different elements of trust are provided by several collaborating parties:
+the identity providers, user home organisations, and research communities. This
+means that trust anchors with varying trust assurance profiles will be
+distributed for use in distinct EGI usage scenarios. Starting mid-2017 (release
+1.82 or above) initially two profiles will be supported. For a discussion on the
+impact, see [EGI document 2745](https://documents.egi.eu/document/2745).
```

Comment:
@dlgroep mid-2017 is gone since some time, is this reference still relevant? Especially as the document 2745 is not a public document.

Reply:
With the combined assurance model (and the associated text now in the HOWTO), I think it's better to drop this extire "## Trust Anchor structure for differentiated (collaborative) assurance" section, also including removal of its subsection "### Notice for sites in EGI that support WLCG". 
The releavnt bits are better placed in the combined assurance model section now.

Reply:
Review comments in-line in the code

Reply:
Thanks, implemented.